tests/InputFiles/ExportEquationOfStateForRotNS/ExportEosForRotNS.yaml

```diff
@@ -0,0 +1,56 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: ExportEquationOfStateForRotNS
+# Check: parse;execute
+# ExpectedOutput:
+#   EOS.GN
+
+EquationOfState:
+  Enthalpy(Enthalpy(Enthalpy(Spectral))):
+    ReferenceDensity: 0.001822038675716006
+    MinimumDensity: 0.001822038675716006
+    MaximumDensity: 0.0034
+    TrigScaling: 0.0
+    PolynomialCoefficients:
+      [1.2730598378768818, 1.2730598378768818, 0.6365299189384409,
```

Comment:
Could you add a few notes here where you got the numbers from that you put into this input file? Maybe link a paper or just say "these are some arbitrary numbers", just so we remember where they come from.